app/assets/javascripts/discourse/tests/integration/components/form-kit/field-test.gjs

```diff
@@ -111,6 +111,10 @@ module("Integration | Component | FormKit | Field", function (hooks) {
     assert.form().hasErrors({ foo: ["Required"], bar: ["Required"] });
     assert.form().field("foo").hasError("Required");
     assert.form().field("bar").hasError("Required");
+
+    await fillIn("input[name=foo]", "0");
+    await formKit().submit();
+    assert.form().field("foo").hasNoError();
```

Comment:
I would prefer we add a input-number-test.gjs file for this please

Reply:
how about put it into [input-test.gjs](https://github.com/discourse/discourse/blob/8d0b62c34446d96d4a98c7fa1c99a4098245fd2c/app/assets/javascripts/discourse/tests/integration/components/form-kit/controls/input-test.gjs)?

Reply:
I would prefer input-number-test.gjs

Reply:
And maybe I should just rename input-test.gjs to input-text-test.gjs